docs/readme/readme.rst

```diff
@@ -10,13 +10,19 @@ ensure a smooth upgrade::
     $ aws logout
     $ pip uninstall awscli-login awscli
 
-Before installing ``awscli-login`` you need to install `AWS CLI`_ V1 or V2.
-Instructions for installing V2 can be found on Amazon's website `here`_:
+Before installing ``awscli-login`` you need to install `AWS CLI`_
```

Comment:
As this step is necessary before installing ``awscli-login``, perhaps this directive should be right after the `Installation` header (at about line 6 in this file)?

Reply:
As it is written, I ask them to remove `awscli` in the first step, then reinstall to the latest V1 or V2. `awscli` use to be a dependency. It is no longer. I could rework it so that is not highlighted but it might make it more complex. I am open to further suggestions, but I am marking this a resolved for now.